Builds/containers/gitlab-ci/verify_head_commit.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env sh
 set -ex
 apt -y update
+DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
```

Comment:
I think the `DEBIAN_FRONTEND=noninteractive` is only required when you pipe the output to another program, but granted, it won't hurt anything.